lms/djangoapps/instructor_task/tasks.py

```diff
@@ -298,3 +299,39 @@ def export_ora2_data(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('generated')
     task_fn = partial(upload_ora2_data, xmodule_instance_args)
     return run_main_task(entry_id, task_fn, action_name)
+
+@task(base=BaseInstructorTask)  # pylint: disable=not-callable
```

Comment:
can we create a generic function that does this tasks and accepts the action name like `section_report` or `assignment_type_report`?